third_party/gpus/cuda_configure.bzl

```diff
@@ -691,16 +691,18 @@ def _get_cuda_config(repository_ctx):
 
     is_windows = _is_windows(repository_ctx)
     cuda_version = config["cuda_version"].split(".")
-    cuda_version = ("64_%s%s" if is_windows else "%s.%s") % (cuda_version[0], cuda_version[1])
-    cudnn_version = ("64_%s" if is_windows else "%s") % config["cudnn_version"]
-
+    
     # cuda_lib_version is for libraries like cuBLAS, cuFFT, cuSOLVER, etc.
     # It changed from 'x.y' to just 'x' in CUDA 10.1.
     if (int(cuda_version[0]), int(cuda_version[1])) >= (10, 1):
         cuda_lib_version = ("64_%s" if is_windows else "%s") % cuda_version[0]
     else:
         cuda_lib_version = cuda_version
```

Comment:
Oops, sorry -- my fault.

But now you are not adding the 64_ prefix to cuda_lib_version for CUDA <10.1. Maybe it would be better to use separate variables for the version list and the version string.